Modify Merge Message

dawelter2 opened the pull request:
Modify merge message to be more friendly and easy to know which branch it was merged into.

* Modify returned message with merged branch by project
* Update Tests to expect right message

Commit:
Modify Merge Message

Modify merge message to be more friendly and easy to know which branch it was merged into.

* Modify returned message with merged branch by project
* Update Tests to expect right message

nicoddemus reviewed (approved):
As we discussed, please add a CHANGELOG entry and then we can merge. 👍

Commit:
Modify Merge Message

Modify merge message to be more friendly and easy to know which branch it was merged into.

* Modify returned message with merged branch by project
* Update Tests to expect right message
* Updated Version to 1.3.0
